lib/syntax_tree/node.rb

```diff
@@ -6463,17 +6387,19 @@ def deconstruct_keys(_keys)
         statements: statements,
         consequent: consequent,
         location: location,
+        beginning: beginning,
         comments: comments
       }
     end
 
     def format(q)
-      ConditionalFormatter.new("if", self).format(q)
+      ConditionalFormatter.new(beginning, self).format(q)
```

Comment:
I mentioned this elsewhere, but I think we should check if it's an elsif here and do that formatting here, otherwise call into `ConditionalFormatter`